rsocket-core/src/main/java/io/rsocket/resume/ServerRSocketSession.java

```diff
@@ -45,6 +48,12 @@
   final ByteBufAllocator allocator;
   final boolean cleanupStoreOnKeepAlive;
 
+  final Queue<Runnable> connectionsQueue;
```

Comment:
This looks like it is meant to allow multiple concurrent attempts to resume but I'm not sure. It would be useful to clarify the case in an implementation comment.

Reply:
added docs